src/geode/model/helpers/convert_to_mesh.cpp

```diff
@@ -635,6 +962,57 @@ namespace geode
         return std::make_tuple( std::move( surface ), std::move( solid ) );
     }
 
+    std::tuple< std::unique_ptr< EdgedCurve< 2 > >, ModelToMeshMappings >
+        new_convert_section_into_curve( const Section& section )
+    {
+        return new_convert_model_into_curve( section );
+    }
+
+    std::tuple< std::unique_ptr< EdgedCurve< 3 > >, ModelToMeshMappings >
+        new_convert_brep_into_curve( const geode::BRep& brep )
+    {
+        return new_convert_model_into_curve( brep );
+    }
+
+    std::tuple< std::unique_ptr< SurfaceMesh3D >, ModelToMeshMappings >
+        new_convert_brep_into_surface( const BRep& brep )
+    {
+        return new_convert_model_into_surface( brep );
+    }
+
+    std::tuple< std::unique_ptr< SurfaceMesh2D >, ModelToMeshMappings >
+        new_convert_section_into_surface( const Section& section )
+    {
+        return new_convert_model_into_surface( section );
+    }
+
+    std::tuple< std::unique_ptr< SolidMesh3D >, ModelToMeshMappings >
+        new_convert_brep_into_solid( const BRep& brep )
+    {
+        std::vector< std::reference_wrapper< const geode::SolidMesh3D > >
+            meshes;
+        meshes.reserve( brep.nb_blocks() );
+        for( const auto& block : brep.blocks() )
+        {
+            meshes.emplace_back( block.mesh() );
+        }
+        auto mesh = geode::detail::create_mesh< SolidMesh3D >( meshes );
+        auto mesh_builder = geode::SolidMeshBuilder< 3 >::create( *mesh );
+        ModelToMeshMappings brep2mesh;
+        for( const auto unique_vertex :
+            geode::Range{ brep.nb_unique_vertices() } )
+        {
+            const auto new_vertex_index = mesh_builder->create_vertex();
+            brep2mesh.unique_vertices_mapping.map(
+                unique_vertex, new_vertex_index );
+        }
+        build_polyhedra_from_model( brep, *mesh_builder, brep2mesh );
```

Comment:
mesh_builder->compute_polydron_adjacencies ?
In order to make sure the adjacencies between different initial blocks are set